### DISCUSSION TITLE
[Feature Request]: 现在的下载目录，和媒体目录，可以增加一个比如叫私人媒体的下载目录，然后对应一个私人媒体电影或者剧集，在下载的时候增加一个右键选项，手动指定下载到对应的目录呢。

### DISCUSSION THREAD

**Discussion by xwndo** [jxxghp/MoviePilot#3473] [讨论]
### 当前程序版本

v1.9.1-1

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

以前nastool时代的时候是可以手动选择目录的，MP时代就取消了，导致现在需要设置一个独立媒体库来使用，只能通过部署2个MP。
看到现在支持这种媒体库设置，可否在增加一个选项来解决这个问题呢。可以在不改变现在的使用体验来来添加一个这个功能，比如直接点击还是下载到默认的，右键就增加一个选项。WX和TG这些搜索也可以增加一个指令比如，搜索 ”xxx电影/剧集 到私人媒体库“ ，这样的指令来下载到私人媒体库。

私人媒体库可以设置一个独立的目录，然后里面也可以使用二级分类这样，让系统自动分类

![image](https://github.com/jxxghp/MoviePilot/assets/44762565/ebfeb827-772b-4092-85ba-7a509f6ccf11)

![截图_20240525181039](https://github.com/jxxghp/MoviePilot/assets/44762565/2a1bda93-9e9a-4382-84cb-7ac9a4551f78)

**Comment by SchrodingerBarbatos** [comment]
我也觉得现在下载目录和媒体库目录这么灵活的情况下，资源下载可以在每个资源右上角三个点除了查看详情也可以添加选择下载目录这个选项
